### ISSUE TITLE
Approvals and Checks for Service Connections

### ISSUE THREAD

**Issue by sandrom** [microsoft/terraform-provider-azuredevops#459]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

I want to be able to set Approvals and Checks for Service Connections like possible in the UI

<img width="707" alt="Screenshot 2021-10-04 at 15 28 06" src="https://user-images.githubusercontent.com/496479/135859865-4214dad9-6143-4261-8faf-ffa6b4af2cfd.png">

Is this a planned feature or am I missing something?

**Comment by xuzhang3** [comment]
@sandrom Blocked by SDK v6

**Comment by sandrom** [comment]
@xuzhang3 so this will come eventually? :)

**Comment by xuzhang3** [comment]
> @xuzhang3 so this will come eventually? :)

yes, you can track progress in https://github.com/microsoft/terraform-provider-azuredevops/issues/459#issuecomment-939771729

**Comment by lindhe** [comment]
I think this is a duplicate of #451, yes?

**Comment by xuzhang3** [comment]
> I think this is a duplicate of #451, yes?

@lindhe yes

**Comment by lindhe** [comment]
> > @xuzhang3 so this will come eventually? :)
> 
> yes, you can track progress in https://github.com/microsoft/terraform-provider-azuredevops/issues/459#issuecomment-939771729

You linked the wrong comment here. :)

**Comment by xuzhang3** [comment]
@lindhe  😄 v6 SDK has release: https://github.com/microsoft/azure-devops-go-api/releases/tag/azuredevops%2Fv6.0.0

**Comment by lindhe** [comment]
Cool! So this and #451 are not blocked anymore then?

**Comment by xuzhang3** [comment]
Bad luck, released version is V6 not V6.1

**Comment by lindhe** [comment]
So it's really blocked by SDK v6.1, not SDK v6?

Is there a timeplan for SDK v6.1?

**Comment by xuzhang3** [comment]
Some news I got is that they will release v7 instead of v6.1 but not ETA for now

**Comment by aristosvo** [comment]
Please vote on this issue: https://developercommunity.visualstudio.com/t/Update-of-the-Go-SDK-to-include-new-feat/10186787

**Comment by xuzhang3** [comment]
`azuredevops_check_approval` supported in #728
